examples/cpp/pcapReader/pcapReader.cpp

```diff
@@ -39,10 +39,20 @@ void pcapHandler (u_char* args,
     }
     else
     {
-        std::cout << "Recieved  : " << ctime((const time_t*)&hdr->ts.tv_sec);
-        std::cout << "Micro Sec : " << hdr->ts.tv_usec <<std::endl;
+        time_t nowtime;
+        struct tm* nowtm;
+        char tmbuf[64], buf[64];
+
+        nowtime = hdr->ts.tv_sec;
+        nowtm = localtime(&nowtime);
+        strftime(tmbuf, sizeof tmbuf, "%Y-%m-%d %H:%M:%S", nowtm);
```

Comment:
This requires a WIN32 #define as %d must be %0d on WIN. Check utils/src/utils.cpp:utils_getUtcDateTime for an example